stages/ui.cc

```diff
@@ -326,10 +340,17 @@ void Ui::UpdateLEDs() {
         }
         uint8_t type = configuration & 0x3;
         LedColor color = palette_[type];
+        uint16_t* seg_config = settings_->mutable_state()->segment_configuration;
         if (settings_->in_seg_gen_mode()) {
-          brightness = chain_state_->loop_status(i) == ChainState::LOOP_STATUS_SELF ?
-            lfo_patterns[configuration >> 8 & 0x3] : fade_patterns[chain_state_->loop_status(i)];
```

Comment:
This stuff still needs to be here or else loop group ends won't oscillate correctly. That also means that you can't just overwrite it, but scale it; that is, get a number where you can `*=` the brightness.

Reply:
Totally forgot to handle the else case for `fade_patterns[chain_state_->loop_status(i)]`, thanks!

Reply:
Should be fixed in https://github.com/qiemem/eurorack/pull/30/commits/45baa659733cdf7125dd15c8355db40c87bbbdcb